packages/app-project/stores/Project.js

```diff
@@ -0,0 +1,31 @@
+import { flow, getRoot, types } from 'mobx-state-tree'
+import _ from 'lodash'
+import { panoptes } from '@zooniverse/panoptes-js'
+import asyncStates from './asyncStates'
+
+const Project = types
+  .model('Project', {
+    displayName: types.optional(types.string, ''),
+    error: types.optional(types.frozen, null),
+    id: types.optional(types.string, ''),
+    state: types.optional(types.enumeration('state', asyncStates), 'initialised')
+  })
+
+  .actions(self => ({
+    fetch: flow(function * fetch (slug) {
+      self.state = 'loading'
+      const { client } = getRoot(self)
+      try {
+        const project = yield client.get('/projects', { slug })
```

Comment:
Should this be `panoptes.get()`? You could also import the project set of request helpers:
```
import { projects } from '@zooniverse/panoptes-js'`
projects.get({ query: { slug }})
```

Reply:
I just noticed that the panoptes client is being set at the store initialization as `client`. So that makes the import here redundant, but also makes the helper functions I wrote for the resource not being used either. If they're not useful, perhaps a different approach should be taken?

Reply:
Yeah, I'm using the MST [dependency injection](https://github.com/mobxjs/mobx-state-tree#dependency-injection) feature to make it easier to mock the transport layer in testing. There's no reason we can't use the helper functions here - I'll add them in.

Reply:
👍 I wanted to check in case they weren't useful in practice. Sounds good.